.eslintrc

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Please use the file from [prod-module-boilerplate](https://github.com/ericelliott/prod-module-boilerplate/blob/master/.eslintrc)